architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,86 @@
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization determines whether an identity is allowed to perform any action.
+Note that authorization is a separate step from
+link:authentication.html[authentication], which is more about determining the
+identity of who is taking the action.
+
+Authorization is managed using:
+
+[horizontal]
+Rules:: Describe sets of permitted verbs on a set of resource types.
+Roles:: Collection of rules.
+Bindings:: Associate users and/or groups with a role.
+
+Users and groups may be bound to multiple roles at the same time.
+
+Several factors are combined to make the decision when authorization is
+evaluated:
+
+[horizontal]
+Identity:: Both the username and list of groups the user belongs to.
+Action:: The action being performed. In most cases, this consists of:
+Project::: The project being accessed.
+Verb::: Can be `get`, `list`, `create`, `update`, or others.
+Resource::: The API endpoint being accessed. This is distinct from the resource
+being touched, such as pods, `*deploymentconfigs*`, or builds.
+Bindings:: The full list of bindings.
+
+The evaluation goes through the following steps:
+
+. Use the identity and the `action.project` to find all bindings that apply to the user or their groups.
+. Use the bindings to locate all the roles that apply.
+. Use the roles to find all the rules that apply.
+. Check the action against each rule to find a match.
+. If no matching rule is found, the action is then denied by default.
+
+You can link:../../admin_guide/manage_authorization_policy.html[manage
+authorization policy] using the administrator CLI.
```

Comment:
s/manage authorization policy/manage authorizations policies (or manage the authorization policy)